central/complianceoperator/v2/profiles/datastore/datastore_impl.go

```diff
@@ -3,129 +3,98 @@ package datastore
 import (
 	"context"
 
-	"github.com/pkg/errors"
 	profileSearch "github.com/stackrox/rox/central/complianceoperator/v2/profiles/datastore/search"
-	edge "github.com/stackrox/rox/central/complianceoperator/v2/profiles/profileclusteredge/store/postgres"
 	pgStore "github.com/stackrox/rox/central/complianceoperator/v2/profiles/store/postgres"
 	v1 "github.com/stackrox/rox/generated/api/v1"
 	"github.com/stackrox/rox/generated/storage"
 	pgPkg "github.com/stackrox/rox/pkg/postgres"
 	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stackrox/rox/pkg/sac/resources"
 	"github.com/stackrox/rox/pkg/search"
-	"github.com/stackrox/rox/pkg/uuid"
 )
 
 var (
 	complianceSAC = sac.ForResource(resources.Compliance)
 )
 
 type datastoreImpl struct {
-	db               pgPkg.DB
-	store            pgStore.Store
-	profileEdgeStore edge.Store
-	searcher         profileSearch.Searcher
+	db       pgPkg.DB
+	store    pgStore.Store
+	searcher profileSearch.Searcher
+}
+
+// ProfileSummary represents a summary of profile data
+type ProfileSummary struct {
```

Comment:
Are we doing this so that the UI doesn't need to make a call to get all profiles for a cluster, then make a subsequent call for `v2/compliance/profiles/{id}`?

Reply:
basically normalzing by query.  Gives the UI the snippet of information for the select screen when building a schedule.

Reply:
I'm not very familiar with the inner workings of the UI, yet. But is the preferred approach to supply summary endpoints instead of calculating them on the front end?